components/orderableTable/OrderableTable.scss

```diff
@@ -0,0 +1,18 @@
+.orderableHelper {
+    z-index: 666;
```

Comment:
😈

Reply:
😄 it just so happened that 665 is z-index of the modal.